From beff9a11604bf08f94144f7eb3e3a9ae804a7ca9 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Mon, 29 Oct 2018 09:29:45 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb: npu voltage regulator use
 tcs4525

Since tcs4525 have better power ripple than syr837, so use tcs4525 as
npu voltage regulator.

Change-Id: Iefaa88f0ee8da6bc67b08fe611d6d037160dd588
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index e58d96cf7d15..8c5b5b334d4f 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -270,9 +270,9 @@
 	status = "okay";
 	clock-frequency = <400000>;
 
-	vdd_npu: syr837@40 {
-		compatible = "silergy,syr827";
-		reg = <0x40>;
+	vdd_npu: tcs4525@1c {
+		compatible = "tcs,tcs452x";
+		reg = <0x1c>;
 		vin-supply = <&vcc5v0_sys>;
 		regulator-compatible = "fan53555-reg";
 		pinctrl-0 = <&vsel_gpio>;
@@ -280,7 +280,7 @@
 		regulator-name = "vdd_npu";
 		regulator-min-microvolt = <750000>;
 		regulator-max-microvolt = <950000>;
-		regulator-ramp-delay = <1000>;
+		regulator-ramp-delay = <2300>;
 		fcs,suspend-voltage-selector = <0>;
 		regulator-boot-on;
 		regulator-state-mem {
-- 
2.35.3

